N/NOMAD/bundled/patches/nomad_openmp.patch

```diff
@@ -6,14 +6,14 @@
  
 +if(CMAKE_C_COMPILER_ID MATCHES "Clang\$")
 +    set(OpenMP_C_FLAGS "-Xpreprocessor -fopenmp")
-+    set(OpenMP_C_LIB_NAMES "omp")
-+    set(OpenMP_omp_LIBRARY omp)
++    set(OpenMP_C_LIB_NAMES "gomp")
++    set(OpenMP_omp_LIBRARY gomp)
```

Comment:
`OpenMP_gomp_LIBRARY` ?

Reply:
Mind opening a PR so that we can quickly see what happens?